tests/test_bar_data.py

```diff
@@ -1153,3 +1162,20 @@ def test_can_trade_restricted(self):
                 restrictions=rlm
             )
             self.assertEqual(bar_data.can_trade(self.ASSET1), info[1])
+
+    def test_can_trade_delisted(self):
+        """
+        Test that can_trade returns False for an asset on or after its auto
+        close date.
+        """
+        dates_to_check = [
+            (pd.Timestamp('2016-01-05', tz='UTC'), True),
```

Comment:
Related to `https://github.com/quantopian/zipline/pull/1644/files#r95641660`, we should add tests that use minute dts.